packages/core/src/components/BubbleList/index.vue

```
@@ -51,91 +59,89 @@ const lastScrollTop = ref(0);
-    );
-    // 如果页面上有监听节点，先移除
-    if (resizeObserver.value) {
-      resizeObserver.value.disconnect();
-    }
-    const lastItem = listBubbles[listBubbles.length - 1];
-    if (lastItem) {
-      resizeObserver.value = new ResizeObserver(() => {
-        if (!stopAutoScrollToBottom.value) {
-          scrollToBottom();
+  const container = checkViewRef.value;
+  if (!container) return;
```

ESLint

[error] Expect newline after if

(antfu/if-newline)